topologies/binding/binding.go

```diff
@@ -165,6 +168,30 @@ func (d *staticDUT) DialCLI(ctx context.Context, opts ...grpc.DialOption) (bindi
 	return newCLI(sc)
 }
 
+func (a *staticATE) DialGNMI(ctx context.Context, opts ...grpc.DialOption) (gpb.GNMIClient, error) {
+	if a.dev.Gnmi == nil {
+		return nil, fmt.Errorf("gnmi must be configured in ate binding")
+	}
+	addr := a.dev.Gnmi.Target
+	opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: true}))) // NOLINT
```

Comment:
Whether to use skip_verify should observe the dial options in the proto. The dialer.dialGRPC already implements the switching in the proper way.

Reply:
Thanks, taken care of.